### PR TITLE
doctrine/doctrine-bundle integration

### PR DESCRIPTION
**Pull request by kreemer**
This PR does some refactorings on the internal health and information system. It also adds an builtin health check and info endpoint for the database connection.

### CODE REVIEW

**Comment by kreemer** [comment]
@akondas could you take a look?

File: composer.json [hunk 1]
```diff
@@ -58,7 +61,7 @@
             "phpunit"
         ],
         "phpstan": [
-            "phpstan analyse"
+            "phpstan analyse --memory-limit=256M"
```
Comment (akondas) [inline composer.json:64]: why?

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -15,33 +15,71 @@ public function getConfigTreeBuilder(): TreeBuilder
...
+                                        ->scalarNode('path')->defaultValue('%kernel.project_dir%')->end()
+                                    ->end()
+                                ->end()
+                                ->arrayNode('database')
+                                    ->addDefaultsIfNotSet()
+                                    ->canBeDisabled()
+                                    ->children()
+                                        ->arrayNode('connections')
+                                            ->useAttributeAsKey('name')
+                                            ->defaultValue(['default' => ['service' => 'Doctrine\DBAL\Connection', 'check_sql' => 'SELECT 1']])
```
Comment (akondas) [inline src/DependencyInjection/Configuration.php:40]: can we add this to readme also (with some examples?)

**Review by akondas — COMMENTED** [review]
looks good, please add some examples and description to readme and we are good to go :wink:

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (kreemer) [reply src/DependencyInjection/Configuration.php:40]: I've updated the `README` and outlined how you can specify the connection service aliases. I hope you find it sufficient. I would like to propose to update the README if we add more components (i'm planning to add the mailer and messager component soon)

File: composer.json (hunk 1 above)
Comment (kreemer) [reply composer.json]: I've removed this line, was for debugging purposes on my local machine

Comment (akondas) [reply composer.json]: thanks

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (akondas) [reply src/DependencyInjection/Configuration.php:40]: :heart_eyes:

File: README.md [hunk 3]
```diff
@@ -52,6 +52,13 @@ Production-ready features for your Symfony application. Actuator endpoints let y
         "SecurityBundle": "Symfony\\Bundle\\SecurityBundle\\SecurityBundle",
         "ActuatorBundle": "Akondas\\ActuatorBundle\\ActuatorBundle"
       }
+    },
+    "database": {
+      "default": {
+        "type": "stgreSQL100",
```
Comment (akondas) [inline README.md:58]: you mean `postgreSQL100`?

File: composer.json [hunk 4]
```diff
@@ -18,19 +18,22 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "symfony/framework-bundle": "^5.2 || ^6.0"
+        "php": ">7.4",
```
Comment (marmichalski) [inline composer.json:21]: Is the PHP 7.4 support dropping intentional?

File: README.md (hunk 3 above)
Comment (kreemer) [reply README.md:58]: I've copied the output of my dev setup.. I assume it's an internal identifier

File: composer.json (hunk 4 above)
Comment (kreemer) [reply composer.json:21]: Sry typo, i will fix it before merging

**Review by akondas — APPROVED** [review]

**Comment by akondas** [comment]
Thanks :beers:

**Pull request merged**